lib/twitter_cldr/js/compiler.rb

```diff
@@ -22,50 +22,21 @@ def initialize(options = {})
         @source_map = options[:source_map]
       end
 
-      def compile_each(options = {})
+      def compile_bundle(bundle, bundle_elements, bundle_hash, options = {})
         options[:minify] = true unless options.include?(:minify)
 
-        @locales.each do |locale|
-          contents = ""
-
-          @features.each do |feature|
-            renderer_const = implementation_renderers[feature]
-            contents << renderer_const.new(:locale => locale, :prerender => @prerender).render if renderer_const
-          end
-
-          bundle = TwitterCldr::Js::Renderers::Bundle.new
-          bundle[:locale] = locale
-          bundle[:contents] = contents
-          bundle[:source_map] = @source_map
-
-          result = CoffeeScript.compile(bundle.render, {
-            :bare => false,
-            :sourceMap => @source_map
-          })
-
-          file = if @source_map
-            CompiledFile.new(result["js"], result["sourceMap"])
-          else
-            CompiledFile.new(result)
+        contents = ""
+        bundle_elements.each do |bundle_element|
+          renderer_const = bundle_hash[bundle_element]
```

Comment:
These two statements can be combined on one line: `if renderer_const = bundle_hash[bundle_element]`. This works because the result of the assignment is returned by `=`, which is true for constants and false for `nil`.